egs/wsj/s5/steps/nnet3/dump_bottleneck_features.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+#           2016  Pegah Ghahremani
+# Apache 2.0
+
+
+# Begin configuration section.
+feat_type=
+stage=1
+nj=4
+cmd=run.pl
+use_gpu=false
+bnf_name=renorm4
+use_ivector=false
+ivector_dir=
+# Begin configuration.
+transform_dir=
+# End configuration options.
+
+echo "$0 $@"  # Print the command line for logging
+
+[ -f path.sh ] && . ./path.sh # source the path.
+. parse_options.sh || exit 1;
+
+if [ $# != 5 ]; then
+   echo "usage: steps/nnet3/dump_bottleneck_features.sh <input-data-dir> <output-data-dir> <bnf-nnet-dir> <archive-dir> <log-dir>"
+   echo "e.g.:  steps/nnet3/dump_bottleneck_features.sh data/train data/train_bnf exp_bnf/bnf_net mfcc exp_bnf/dump_bnf"
+   echo "main options (for others, see top of script file)"
+   echo "  --config <config-file>                           # config containing options"
+   echo "  --nj <nj>                                        # number of parallel jobs"
+   echo "  --cmd (utils/run.pl|utils/queue.pl <queue opts>) # how to run jobs."
+   echo "  --ivector-dir                                    # directory for ivectors"
+   exit 1;
+fi
+
+data=$1
+bnf_data=$2
+nnetdir=$3
+archivedir=$4
+dir=$5
+
+# Assume that final.nnet is in nnetdir
+cmvn_opts=`cat $nnetdir/cmvn_opts`;
+bnf_nnet=$nnetdir/final.raw
+if [ ! -f $bnf_nnet ] ; then
+  echo "No such file $bnf_nnet";
+  exit 1;
+fi
+
+if $use_gpu; then
+  train_queue_opt="--gpu 1"
+  prior_gpu_opt="--use-gpu=yes"
+  if ! cuda-compiled; then
+    echo "$0: WARNING: you are running with one thread but you have not compiled"
+    echo "   for CUDA.  You may be running a setup optimized for GPUs.  If you have"
+    echo "   GPUs and have nvcc installed, go to src/ and do ./configure; make"
+    exit 1
+  fi
+else
+  echo "$0: without using a GPU this will be very slow.  nnet3 does not yet support multiple threads."
+  parallel_train_opts="--use-gpu=no"
+  prior_gpu_opt="--use-gpu=no"
+  prior_queue_opt=""
+fi
+
+
+## Set up input features of nnet
+if [ -z "$feat_type" ]; then
```

Comment:
I think LDA feats are no longer supported in the scripts. Check the other nnet3 training scripts to see if this part is necessary and remove the LDA and delta feats sections if not necessary.